api/src/main/java/com/lifegraph/team20/controller/LifeGraphsContorller.java

```diff
@@ -0,0 +1,55 @@
+package com.lifegraph.team20.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.lifegraph.team20.models.Content;
+import com.lifegraph.team20.service.ContentService;
+
+@RestController
+@RequestMapping("/life-graph")
+public class LifeGraphsContorller {
+    @Autowired
+    ContentService contentService;
+
+    @RequestMapping(value = "/new",method = RequestMethod.POST)
```

Comment:
```suggestion
    @PostMapping
```
同じURLでHTTPメソッドを変更することで、エントリポイントとなるコントローラのメソッドを変更できます。